db/db-tests.md

```diff
@@ -0,0 +1,40 @@
+This contains sections describing how to setup data in mongodb to exercise code as it is developed for dp-topic-api
```

Comment:
Could this set up be scripted? I.e. write a simple go script that uses the driver to create the necessary collections / documents? Could also create a tear down script to clean up.

Reply:
Not worth the effort, that would consume way too much time ... and we don't do integration testing, so this is a good alternative.
Have you tried the script in the instructions (Dave Subiros managed to follow test instructions just fine for previous PR: https://github.com/ONSdigital/dp-topic-api/pull/19) ?

Reply:
We don't do integration testing yet - but there is/has been technical discussion of how we can implement it into the CI process.